src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -114,6 +114,7 @@ public boolean compile(String cp) throws IOException, InterruptedException {
         }
         Path gvmPath = paths.getGvmPath();
         FileOps.rmdir(paths.getTmpPath());
+        Files.createDirectories(paths.getTmpPath());
```

Comment:
Don't we have generic code that makes sure the required paths are created?

Reply:
We do, but the line before just removes that path